lib/notify/email.ex

```diff
@@ -0,0 +1,71 @@
+defmodule Notify.Email do
+  @moduledoc """
+  This module holds the email notification functionality we use when updating users of various occurences in the system
+  """
+  import Bamboo.Email
+  import Bamboo.SendGridHelper
+
+  @doc """
+  This function builds an email that informs a user of their attempted changes being approved and applied
+  """
+  @spec change_approved(map()) :: Bamboo.Email.t()
+  def change_approved(change) do
+    temp_id = System.get_env("SENDGRID_APPROVED_TEMPLATE_ID")
```

Comment:
Since `temp` _typically_ stands for "temporary" I think it is best we go with `template_id` here.

Reply:
:D